src/array.js

```diff
@@ -4,176 +4,176 @@
 
 // cons :: a -> [a] -> [a]
 // a with x prepended
-export function cons(x, a) {
-    const l = a.length;
-    const b = new Array(l + 1);
-    b[0] = x;
-    for (let i = 0; i < l; ++i) {
-        b[i + 1] = a[i];
-    }
-    return b;
+export function cons (x, a) {
+  const l = a.length
+  const b = new Array(l + 1)
+  b[0] = x
+  for (let i = 0; i < l; ++i) {
+    b[i + 1] = a[i]
+  }
+  return b
 }
 
 // append :: a -> [a] -> [a]
 // a with x appended
-export function append(x, a) {
-    const l = a.length;
-    const b = new Array(l + 1);
-    for (let i = 0; i < l; ++i) {
-        b[i] = a[i];
-    }
-
-    b[l] = x;
-    return b;
+export function append (x, a) {
+  const l = a.length
+  const b = new Array(l + 1)
+  for (let i = 0; i < l; ++i) {
+    b[i] = a[i]
+  }
+
+  b[l] = x
+  return b
 }
 
 // drop :: Int -> [a] -> [a]
 // drop first n elements
-export function drop(n, a) {
-    if (n < 0) {
-        throw new TypeError('n must be >= 0');
-    }
+export function drop (n, a) { //eslint-disable-line
```

Comment:
Oh drat, I forgot jshint can do that but eslint can't :(

It looks like it might be possible to do disable a specific rule on a specific line, like: `// eslint-disable-line complexity` or `// eslint-disable-next-line complexity`.  Will that work?